website/docs/d/iam_policy_document.html.markdown

```diff
@@ -84,10 +84,21 @@ The following arguments are supported:
   current policy document.  Statements with non-blank `sid`s in the current
   policy document will overwrite statements with the same `sid` in the source
   json.  Statements without an `sid` cannot be overwritten.
+* `source_json_list` (Optional) - A list of IAM policy documents to import as
+  a base for the current policy document. Statements across all policy documents
+  defined in this list or in the `source_json` attribute must be unique. Statements
+  with non-blank `sid`s in the current policy document will overwrite statements
+  with the same `sid` from any of the source json documents.
 * `override_json` (Optional) - An IAM policy document to import and override the
   current policy document.  Statements with non-blank `sid`s in the override
-  document will overwrite statements with the same `sid` in the current document.
+  document will overwrite statements with the same `sid` in the current document,
+  including any defined by the `override_json_list` attribute.
   Statements without an `sid` cannot be overwritten.
+* `override_json_list` (Optional) - A list of IAM policy documents to import and
```

Comment:
Open to suggestions around the documentation for this attribute. I found it quite hard to find a simple way to describe the iterative merge based on the ordering of the array...